app/components/SearchBox.tsx

```diff
@@ -34,24 +34,23 @@ const SearchBox: React.FunctionComponent<Props> = ({
     type: 'object',
     properties: {
       searchField: {
+        title: 'Search By:',
         type: 'string',
         enum: dropdownSortItems
       },
       searchValue: {
+        title: 'Search Term:',
         type: 'string'
       }
     }
   };
 
   const uiSchema = {
     searchField: {
-      'ui:placeholder': 'Search By: ',
```

Comment:
When things are labelled by using the input's `placeholder` attribute, this isn't recognizable as a label by a screenreader. The best practice is to use a `<label>` element as shown above.